configure

```diff
@@ -606,11 +615,12 @@ def configure_arm(o):
   else:
     arm_float_abi = 'default'
 
+  o['variables']['arm_fpu'] = options.arm_fpu
```

Comment:
Does it really make sense to make this explicit setting overridable by the configure script? It seems like if someone is passing the option explicitly, it should not be overridden. For example, what if an armv7 user would want to use vfp or vfpv2 instead of vfpv3 (e.g. for debugging or other reasons)?